### DISCUSSION TITLE
Lightning using pruned Bitcoin Node

### DISCUSSION THREAD

**Discussion by ffrediani** [lightningnetwork/lnd#8792] [General]
Hello

Does anyone know if there are any real issues or concerns in connecting your Lightning Node to a pruned Bitcoin Node ? if so could elaborate a bit about it ?

I understand the Lightning needs to broadcast transactions on chain in order to open channels and to close them and that is done with the latest blocks and older transactions may not be needed (not sure how old).

An important point on it to find out is if there is a reasonable minimum size for a pruned node or if what really matters is the ability to broadcast a new channel opening and close it and not very necessary to access much older transactions.

**Comment by guggero** [comment]
For reference: https://github.com/lightningnetwork/lnd/issues/8250#issuecomment-2138835095

**Comment by ffrediani** [comment]
Thanks @guggero 
Guess best to concentrate the discussion there then.
